data/css/news.scss

```diff
@@ -118,3 +118,10 @@ $support-font: 'Fira Sans' !default;
         }
     }
 }
+
+.WindowSimple {
+    scrollbar {
+        // margin-top is for being covered by Layout.TopMenu
+        margin-top: 50px;
```

Comment:
We need to open a new task to handle this properly.
Ideally this should be the responsibility of the container that packs the topbar over the webview

Reply:
If I understand correctly what you mean, then I agree! Can you make sure to open this issue?